stories/object-identifier/object-identifier.stories.js

```diff
@@ -0,0 +1,173 @@
+</div>
+`;
+
+titleOnly.storyName = 'Title';
+titleOnly.parameters = {
+    docs: {
+        storyDescription: 'To display the object identifier as a title, add the `fd-object-identifier__title` modifier class to the main element.'
+    }
+};
+
+export const boldTitle = () => `
+<div class="fd-object-identifier">
+    <p class="fd-object-identifier__title fd-object-identifier__title--bold">NoteBook Basic 15</p>
+</div>
+`;
+
+boldTitle.storyName = 'Bold';
+boldTitle.parameters = {
+    docs: {
+        storyDescription: `To display the object identifier's title in bold, replace \`fd-object-identifier__title\`
+        with the \`fd-object-identifier__title--bold\` modifier class.`
+    }
+};
+
+export const titleAsLink = () => `
+<div class="fd-object-identifier">
+    <p class="fd-object-identifier__title fd-object-identifier__title--link">NoteBook Basic 15</p>
+</div>
+<br>
+<div class="fd-object-identifier">
+    <p class="fd-object-identifier__title fd-object-identifier__title--bold fd-object-identifier__title--link">NoteBook Basic 15</p>
+</div>
+<br>
+<div class="fd-object-identifier">
+    <p class="fd-object-identifier__title"><a href="#" class="fd-link fd-object-identifier__link">NoteBook Basic 15</a></p>
+</div>
+<br>
+<div class="fd-object-identifier">
+    <p class="fd-object-identifier__title fd-object-identifier__title--bold"><a href="#" class="fd-link fd-object-identifier__link">NoteBook Basic 15</a></p>
+</div>
+`;
+
+titleAsLink.storyName = 'Link';
+titleAsLink.parameters = {
+    docs: {
+        storyDescription: 'To display the object identifier as a link, add the `fd-object-identifier__title--link` modifier class to the title element.'
+    }
+};
+
+export const titleAndText = () => `
+<div class="fd-object-identifier">
+    <p class="fd-object-identifier__title">NoteBook Basic 15</p>
+    <p class="fd-object-identifier__text">Notebook Basic 15 with 2,80 GHz quad core, 15" LCD, 4 GB DDR3 RAM, 500 GB Hard Disc, Windows 8 Pro</p>
+</div>
+
+<br>
+
+<div class="fd-object-identifier">
+    <p class="fd-object-identifier__title fd-object-identifier__title--bold">NoteBook Basic 15</p>
+    <p class="fd-object-identifier__text">Notebook Basic 15 with 2,80 GHz quad core, 15" LCD, 4 GB DDR3 RAM, 500 GB Hard Disc, Windows 8 Pro</p>
+</div>
+
+<br>
+
+<div class="fd-object-identifier">
+    <p class="fd-object-identifier__title fd-object-identifier__title--link">NoteBook Basic 15</p>
+    <p class="fd-object-identifier__text">Notebook Basic 15 with 2,80 GHz quad core, 15" LCD, 4 GB DDR3 RAM, 500 GB Hard Disc, Windows 8 Pro</p>
+</div>
+
+<br>
+
+<div class="fd-object-identifier">
+    <p class="fd-object-identifier__title fd-object-identifier__title--bold fd-object-identifier__title--link">NoteBook Basic 15</p>
+    <p class="fd-object-identifier__text">Notebook Basic 15 with 2,80 GHz quad core, 15" LCD, 4 GB DDR3 RAM, 500 GB Hard Disc, Windows 8 Pro</p>
+</div>
+`;
+
+titleAndText.storyName = 'Description';
```

Comment:
The Fiori 3 docs refer to it as "Descriptive"

Reply:
fixed